src/services/measure.service.js

```diff
@@ -212,7 +212,7 @@ const evaluateMeasure = async (args, { req }) => {
  * Evaluate measure for "population" report type
  * @param {Object} args the args object passed in by the user, includes measure id
  * @param {Object} req http request object
- * @returns {Object} FHIR MeasureReport with population results
+ * @returns {Object} Parameters resource containing one or more Bundles of MeasureReports.
```

Comment:
```suggestion
 * @returns {Object} Parameters resource containing one Bundle with MeasureReports.
```
Just to match the other return definition more closely